src/parser/mod.rs

```diff
@@ -645,6 +645,7 @@ impl<'a> Parser<'a> {
                 Keyword::COMMENT if self.dialect.supports_comment_on() => self.parse_comment(),
                 Keyword::PRINT => self.parse_print(),
                 Keyword::RETURN => self.parse_return(),
+                Keyword::EXPORT => self.parse_export(),
```

Comment:
Could we do [similar to here](https://github.com/apache/datafusion-sqlparser-rs/blob/1a7e8d969ae509d99f5f9aa1136092da002a77c6/src/parser/mod.rs#L607) where we call `prev_token()` so that the `parse-export()` function is stand alone and able to parse a complete statement?